src/TurbulenceClosures/turbulence_closure_diagnostics.jl

```diff
@@ -67,6 +67,25 @@ function cell_diffusion_timescale(closure::AnisotropicBiharmonicDiffusivity, dif
                Δz^2 / max_κz)
 end
 
+function cell_diffusion_timescale(closure::HorizontallyCurvilinearAnisotropicDiffusivity, diffusivities, grid)
+    Δx = min_Δx(grid)
+    Δy = min_Δy(grid)
+    Δz = min_Δz(grid)
+
+    max_νh = maximum_numeric_diffusivity(closure.νh)
+    max_νz = maximum_numeric_diffusivity(closure.νz)
+
+    max_κh = maximum_numeric_diffusivity(closure.κh)
+    max_κz = maximum_numeric_diffusivity(closure.κz)
+
+    return min(Δx^2 / max_νh,
```

Comment:
It occurs to me that if we define something like `max_nuh_kappah` to be the maximum of `max_nuh` and `max_kappaz` then we only need to have 3 lines in the return instead of 6.  Don't imagine this would make any difference but I thought I'd share the thought.

Reply:
That's true! My only response is that sometimes more lines is better if the code is easier to read and understand. But your point still could be worth implementing. Note this change was actually made in #1375 (which this PR depends on)

Reply:
This is a question of style and agree that what you have here works and there is no need to change it.